src/tests/ftest/scripts/setup_nodes.sh

```diff
@@ -37,10 +37,12 @@
 # THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 set -eux
-# using mmap()'ed ULT stacks requires to bump system default
-sudo sysctl vm.max_map_count=1000000
 # allow core files to be generated
 sudo bash -c "set -ex
+# using mmap()'ed ULT stacks requires to bump system default
+cat <<EOF > /etc/sysctl.d/10-daos-server.conf
+vm.max_map_count=1000000
+EOF
```

Comment:
You don't actually need this here.  The RPM installation will install this setting into `/usr/lib/sysctl.d/` on the test nodes.  Having it here means we are not testing the value that the RPM installs.

```suggestion
```

Reply:
sure, and done in next commit.